autoware.repos

```diff
@@ -104,10 +104,16 @@ repositories:
     type: git
     url: https://github.com/tier4/nebula.git
     version: main
+  # Fork of transport_drivers that enables reduction of copy operations
   sensor_component/external/transport_drivers:
     type: git
-    url: https://github.com/MapIV/transport_drivers.git
-    version: boost
+    url: https://github.com/mojomex/transport_drivers
+    version: mutable-buffer-in-udp-callback
+  # Continental compatible version of ROS 2 socket CAN
+  sensor_component/external/ros2_socketcan:
+    type: git
+    url: https://github.com/knzo25/ros2_socketcan
+    version: feat/continental_fd
```

Comment:
```suggestion
  sensor_component/ros2_socketcan:
    type: git
    url: https://github.com/autowarefoundation/ros2_socketcan
    version: feat/continental_fd
```

I've pushed [@knzo25 's commit](https://github.com/knzo25/ros2_socketcan/commit/4ced5284fdd080274b04530406d887e8330a0cdb) to https://github.com/autowarefoundation/ros2_socketcan/tree/feat/continental_fd upstream and created a PR for it on top of main:
- https://github.com/autowarefoundation/ros2_socketcan/pull/50/files

@knzo25 @mojomex could you check if it still works on the latest branch?

Also now since it is in the same organization, we don't need to put it into `external` folder.